translations/mod-record-specifications/en.json

```
@@ -34,5 +34,5 @@
   "validation.missingSubfield": "Subfield ''{code}'' is required.",
   "validation.undefinedSubfield": "Subfield ''{code}'' is undefined.",
   "validation.nonRepeatableSubfield": "Subfield ''{code}'' is non-repeatable.",
-  "validation.invalidLccnSubfieldValue": "LCCN value at 010 $a does not conform to the correct structure."
+  "validation.invalidLccnSubfieldValue": "Fail: 010 $a is in an invalid format."
```

```suggestion
  "validation.invalidLccnSubfieldValue": "010 $a is in an invalid format."
```
`Fail` will be added on UI